test/pdmtypes.jl

```diff
@@ -83,4 +83,30 @@ using Test
         @test d isa PDiagMat{eltype(v),typeof(v)}
         @test d.diag === v
     end
+
+    @testset "division of vectors (dim = 1)" begin
+        A = rand(1, 1)
+        x = randn(1)
+        y = x / A
+        @assert x / A isa Matrix{Float64}
```

Comment:
@devmotion, This is no longer true on 1.9:

```
julia> y = x / A
1-element Vector{Float64}:
 1.1240182465158486
```

Reply:
Do you remember which PR/commit changed this behaviour? I quickly looked through the Julia repo but did not immediately find it.